// let a = 5 
// let b = '/'
// let c = 0

// let ans = 0;

// try{
//     if(c == 0) {
//         throw "You are dividing the value by 0, which is not possble"
//     }
//     if(c > 0) {
//         ans = a/b
//     } 
    

// }
// catch(err) {
//     ans = -1;
//     console.log(err)
// }

// console.log(ans)


try{
    const a = 2;
    a = 3;
    console.log(a);
}
catch(err) {
    console.log("Hello there is an error");
}